index.html

```diff
@@ -316,7 +346,7 @@ <h3>
           <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
         </div>
         <div class="form-container">
-          <form class="contact-form" action="https://formspree.io/f/xzbollge" method="post">
+          <form class="contact-form" id="contact-form" action="https://formspree.io/f/xzbollge" method="post">
             <input name="Full-Name" id="name" type="text" placeholder="Full Name" required><br>
 
             <svg viewBox="0 0 100 10" xmlns="http://www.w3.org/2000/svg">
```

Comment:
- [ ] Kindly make sure your contact section sticks to the Figma design as required for the project.(also remember to use the correct background color)

### Figma design
![Screenshot from 2022-07-16 16-06-34](https://user-images.githubusercontent.com/83042742/179358261-906c1cff-785f-40c2-8a24-760bfaf7963f.png)

### Your design
![Screenshot from 2022-07-16 16-08-24](https://user-images.githubusercontent.com/83042742/179358299-cf246214-75a9-4e95-800d-ea2056464377.png)